lib/inquisition/rails_best_practices/vulnerability.rb

```diff
@@ -0,0 +1,86 @@
+module Inquisition
+  module RailsBestPractices
+    class Vulnerability
```

Comment:
`Vulnerability` - not quite the right name, would be better to rename it to `Issue`, what do you think?